gse-afs-ext-base/src/main/resources/lang/ContingencyStore.properties

```diff
@@ -5,3 +5,7 @@ NewName=New name
 Rename=Rename
 RenameContingency=Rename contingency
 Remove=Remove
+UnsavedQuitConfirmation=is not saved, its modifications will be lost. Would you like to save the file first ?
+SaveAndContinue=Save and Continue
+DontSaveAndContinue=Don't save and Continue
+Cancel=Cancel
```

Comment:
Sort entries by alphabetical order: it will be easier to find the missing properties